mcp/src/index.ts

```diff
@@ -30,13 +30,15 @@ server.registerTool(
   "init_java_sdk",
   {
     description:
-      "Initialize the tsp-location.yaml for generating Java SDK, from URL to tspconfig.yaml. Make sure you ask for the correct url containing commit id, not branch name.",
+      "Initialize the tsp-location.yaml for generating Java SDK, from local path or remote URL to tspconfig.yaml. Always ask for the local path or remote URL. Make sure you ask for the remote URL containing commit id, not branch name.",
     inputSchema: {
-      cwd: z
+      localTspConfigPath: z
         .string()
-        .describe("The absolute path to the current workspace directory"),
+        .optional()
+        .describe("The local path to the tspconfig.yaml file"),
```

Comment:
nit, absolute path?

Reply:
thanks, I will update in this pr: https://github.com/Azure/azure-sdk-for-java/pull/45927